.circleci/config.yml

```
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@0.10.0
+  architect: giantswarm/architect@0.11.0
```

```suggestion
  architect: giantswarm/architect@0.14.0
```
There's 0.14.0 already https://github.com/giantswarm/architect-orb/releases